provisioning_templates/snippet/remote_execution_reboot.erb

```diff
@@ -0,0 +1,16 @@
+<%#
+  kind: snippet
+  name: remote_execution_reboot
+%>
+#
+# Proper reboot of a host. It is not recommended to use "shutdown" command
+# directly as it kills SSH connection and Remote Execution may mark the job
+# as failed which might lead to reexecution in the worst case.
+#
+# Parameters:
+#
+# remote_execution_minutes_to_reboot: number of minutes to reboot (default 1)
+#
+
+<% min = host_param('remote_execution_minutes_to_reboot') || 1 %>
```

Comment:
Does it make sense to do this in seconds? A minute is a long time and I can imagine some users might want to drop to 30 seconds.